bindings/nodejs/lib/wallet/WalletMethodHandler.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 IOTA Stiftung
```

Comment:
```suggestion
// Copyright 2023 IOTA Stiftung
```
But actually I don't know if this is a new file of if GH has some trouble mentioning it's been moved